app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
tìm hiểu về present? nhé
dùng present? thay vì dùng nil? và phải nghịch đảo kết quả

Reply:
present? sẽ trả về giá trị true nếu object trong nil, false, empty hay chỉ toàn kí tự trắng. Hàm này hoàn toàn ngược với blank